<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cadastro_de_clientes</class>
 <widget class="QDialog" name="cadastro_de_clientes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>751</width>
     <height>491</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Dados Clientes</string>
    </attribute>
    <widget class="QPushButton" name="btn_salvar_cliente">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>420</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Salvar Cliente</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_novo_cliente">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Novo Cliente</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>721</width>
       <height>361</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_15">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_16">
          <item>
           <widget class="QLabel" name="label_29">
            <property name="text">
             <string>Nome Fantasia</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_cad_nome_fantasia_cli"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_17">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>CNPJ</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_cad_cnpj_cli"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_18">
          <item>
           <widget class="QLabel" name="label_24">
            <property name="text">
             <string>Razão Social</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_cad_razao_social_cli"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_19">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>CEP</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_cad_cep_cli">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_20">
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Bairro</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_cad_bairro_cli">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_21">
          <item>
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Endereço</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_cad_endereco_cli">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_22">
          <item>
           <widget class="QLabel" name="label_30">
            <property name="text">
             <string>Número</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_cad_numero_cli">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_23">
          <item>
           <widget class="QLabel" name="label_31">
            <property name="text">
             <string>Cidade</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_cad_cidade_cli">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_24">
          <item>
           <widget class="QLabel" name="label_32">
            <property name="text">
             <string>Contato do cliente</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_cad_cont_nome_cli">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_25">
          <item>
           <widget class="QLabel" name="label_33">
            <property name="text">
             <string>Telefone</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_cad_cont_tel_cli">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_26">
          <item>
           <widget class="QLabel" name="label_34">
            <property name="text">
             <string>Ramal</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_cad_cont_ram_cli">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_27">
          <item>
           <widget class="QLabel" name="label_35">
            <property name="text">
             <string>Email</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_cad_cont_email_cli">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_28">
          <item>
           <widget class="QLabel" name="label_36">
            <property name="text">
             <string>Departamento</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_cad_cont_dep_cli">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>Histórico Pedidos</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_7">
    <attribute name="title">
     <string>Histórico Financeiro</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_8">
    <attribute name="title">
     <string>Consultar Clientes</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
